apps/src/dance/blockly/setup.ts

```diff
@@ -0,0 +1,31 @@
+import danceBlocks from '@cdo/apps/dance/blockly/blocks';
+import * as commonBlocks from '@cdo/apps/blocksCommon';
+import {LevelProperties} from '@cdo/apps/lab2/types';
+
+const blockUtils = require('@cdo/apps/block_utils');
+
+// Install common blocks for Dance Party Lab2.
+export function installCommonBlocks(
+  skin: string | undefined,
+  isK1: boolean | undefined
+) {
+  const blockInstallOptions = {
+    skin,
+    isK1,
+  };
+  commonBlocks.install(Blockly, blockInstallOptions);
+  Blockly.setInfiniteLoopTrap();
+}
+
+// Install custom blocks for Dance Party Lab2.
+export function installDanceBlocks(
+  levelProperties: LevelProperties | undefined
```

Comment:
Can we just pass in `sharedBlocks` here? So that the entire level properties object is not a dependency to this function.

Reply:
Great idea - updated  here and in `DanceView`.